.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           path: tools
           key: ${{ runner.os }}-tools-${{ hashFiles('recipe.cake') }}
       - name: Build Addin
-        uses: cake-build/cake-action@v1
+        uses: cake-build/cake-action@v1.1.1
```

Comment:
As per conversation with @ecampidoglio doing this is actually the wrong approach, and should be left as just `@v1` so that all releases under 1.x.x are considered, and the latest version used.